src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset name="fieldset" class="fieldset">
+        <legend name="legend">Personal information</legend>
+        <label class="form-field" name="label">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="input"
+          >
+        </label>
+        <label class="form-field" name="input">
```

Comment:
please add empty line between siblings
```suggestion
        </label>
        
        <label class="form-field" name="input">
```